packages/react/src/Pagination/Pagination.tsx

```diff
@@ -124,30 +127,21 @@ export const Pagination = forwardRef(
     }
 
     return (
-      <nav {...restProps} className={clsx('ams-pagination', className)} ref={ref} aria-label="Paginering">
+      <nav {...restProps} className={clsx('ams-pagination', className)} ref={ref}>
+        <span className="ams-visually-hidden">{ariaLabel}</span>
```

Comment:
This currently isn't read by screen readers. I think you need to connect it to the `nav` with `aria-labelledby`

Reply:
[Yep](https://www.w3.org/TR/WCAG20-TECHS/H97.html#H97-ex2)

Reply:
`aria-labelledby` has been added. Confirmed that NVDA now reads the label.